### PR TITLE
#160663195 Fix Approve and Reject Button Color

### PR DESCRIPTION
**Pull request by albert85**
#### What does this PR do?
Fix approval and reject button color when clicked

#### Description of Task to be completed?
- Fix approve button color when clicked
- Fix reject button colour when clicked
- Fix modal width when notification is clicked

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_front.git
- Change into the project directory
- Run the server `make start`
- Navigate to localhost:3000
- Login as a manager role using your andela account
- Navigate to `My approval` page
- Click on any travel requests to be approved or rejected.
- Click approve or reject button
- The appropriate color displays
- Exit the modal
- Click notification on the notification pane, it should be of the same size with the request modal

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160663195](https://www.pivotaltracker.com/story/show/160663195)

#### Screenshots (if appropriate)
screen shot 2018-09-20 at 8 59 47 pm
screen shot 2018-09-20 at 8 59 56 pm
screen shot 2018-09-20 at 9 22 02 pm

#### Questions:
N/A

### CODE REVIEW

File: src/components/Forms/NewRequestForm/NewRequestForm.jsx [hunk 1]
```diff
@@ -20,11 +20,17 @@ class NewRequestForm extends PureComponent {
 
     this.defaultState = {
       values: {
-        name: !(/^null|undefined$/).test(user) ? user : '', // FIX: need to be refactor later
-        gender: !(/^null|undefined$/).test(gender) ? gender: '',
-        department: !(/^null|undefined$/).test(department) ? department: '',
-        role: !(/^null|undefined$/).test(role) ? role :'',
-        manager: !(/^null|undefined$/).test(manager) ? manager : '',
+        name: user || '', // FIX: need to be refactor later
```
Comment (samuelkubai) [inline src/components/Forms/NewRequestForm/NewRequestForm.jsx]: @albert85 this is the origin of the null fields on the new request form, why did you change them to this format?

Comment (albert85) [reply src/components/Forms/NewRequestForm/NewRequestForm.jsx]: @samuelkubai it was changed when i resolve conflict from collin work merged to develop. let me change it back
